### PR TITLE
[MIG] product_tax_multicompany_default: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
- Update README
- Update tests

cc @Tecnativa

### CODE REVIEW

File: product_tax_multicompany_default/tests/test_product_tax_multicompany.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2018 Vicent Cubells - Tecnativa <vicent.cubells@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from openerp.tests.common import SavepointCase
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/tests/test_product_tax_multicompany.py]: s/openerp/odoo

File: product_tax_multicompany_default/views/product_template_view.xml [hunk 2]
```diff
@@ -6,10 +6,10 @@
     <record model="ir.ui.view" id="product_template_form_view">
         <field name="name">Product template form (multi-company tax button)</field>
         <field name="model">product.template</field>
-        <field name="inherit_id" ref="account.product_template_form_view" />
+        <field name="inherit_id" ref="account.product_template_form_view"/>
         <field name="arch" type="xml">
             <field name ="taxes_id" position="after">
-              <button name="set_multicompany_taxes"
+                <button name="set_multicompany_taxes"
```
Comment (pedrobaeza) [inline product_tax_multicompany_default/views/product_template_view.xml]: Incorrect indentation + align also rest of attributes names

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by cubells** [comment]
@pedrobaeza can you recheck this?

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

File: product_tax_multicompany_default/models/product.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```
Comment (oihane) [inline product_tax_multicompany_default/models/product.py]: The first line # -*- coding: utf-8 -*- is not needed anymore in Python 3

File: product_tax_multicompany_default/tests/test_product_tax_multicompany.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
```
Comment (oihane) [inline product_tax_multicompany_default/tests/test_product_tax_multicompany.py]: It's not needed

**Comment by cubells** [comment]
@oihane done!

**Review by oihane — APPROVED** [review]
Perfect, technically reviewed

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**